isis/src/core/CMakeLists.txt

```diff
@@ -2,8 +2,9 @@ cmake_minimum_required(VERSION 3.10)
 
 project (core VERSION 0.0.1)
 option(buildStaticCore "Build libisis static as well as dynamic"        OFF )
+option (BUILD_CORE_TESTS "Build core module tests" ON)
 
-list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake")
+list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake")
```

Comment:
PROJECT_SOURCE_DIR should be used in sub-modules because it is relative to the last project call, the PVL Core in this case, instead of whichever CMakeLists.txt file was called first.